.env.local.example

```diff
@@ -0,0 +1 @@
+GOOGLE_BOOKS_API_KEY=<YOUR_API_KEY>
```

Comment:
It looks like Overseerr are embedding their API keys directly into their open source project ([reference](https://github.com/sct/overseerr/blob/3c65c08aac68c3379b150769f44bc78e03bf242c/server/api/themoviedb/index.ts#L110C20-L110C51))

Thoughts on that?

Reply:
Seems to open their key for abuse. These APIs are similar (TMDB and GB) as they are free but have limits. We could include a shared one in the docker image (still a leak but offers some protection) or include a temporary one and rely on the users to get their own eventually. Or require them to enter their own key and include detailed steps on how (it is very easy). If we are leaking one then we just have to monitor for abuse and rotate it in that case (and users will need a new key too)

Reply:
The less to maintain the better? I'm for the latter for our convenience but not opposed to the shared one in the docker image if ease of use is the primary concern and not a hassle to maintain.